doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -627,7 +627,9 @@ Mixed references is the default to build when no options are specified. _Note th
 - `--with-mixedrefs=static` (this is the default) create a mixed references build which avoids runtime checks by compiling source twice
 - `--with-noncompressedrefs` create a build supporting non-compressed references only
 
-:pencil: **AArch64 macOS only:** Please specify `--with-noncompressedrefs` because compressed references is not supported on AArch64 macOS yet.
+:pencil: **AArch64 macOS only:**
+  - Please specify `--with-noncompressedrefs` because compressed references is not supported on AArch64 macOS yet.
```

Comment:
While you're here, please fix the typo: This should say "compressed references **_are_** not supported". The same typo is in all versions. Also, the instructions for building Java 8 should have the same comment that only cmake is supported.

Reply:
Fixed the typo.

OpenJ9 for AArch64 macOS does not support Java 8, and I added the comment to the build instructions for V8.